src/main/java/card/issuer/CardIssuerParameters.java

```diff
@@ -0,0 +1,22 @@
+package main.java.card.issuer;
+
+import java.util.ArrayList;
+
+//for now it stores only information about length, but it provides possibility to expand program to check other values
+//like digits in IIN Range, which identifies the Issuer
+public class CardIssuerParameters {
+
+    CardIssuerParameters(String crdIssuerNm, ArrayList<Integer> lngthOfCrdNmb){
```

Comment:
Rename input parameters
```suggestion
    CardIssuerParameters(String cardIssuerName, ArrayList<Integer> lengthsOfCardNumber ){
        this.cardIssuerName = cardIssuerName;
        this.lengthsOfCardNumber = lngthOfCardNumber;
        }
```